### ISSUE TITLE
Virtual Chassis device/interfaces are not grouped

### ISSUE THREAD

**Issue by darcynz** [mattieserver/netbox-topology-views#33]
When modelling connections to a virtual chassis interfaces appear as separate devices on the map. 

Not sure how difficult this would be but, perhaps the switch members of a chassis should be connected by by default as they represent 1 logical device.

**Comment by jpobeda** [comment]
I'm seeing something similar with server chassis where you don't normally document the internal mapping of the virtual nics and I end up having all these blades floating around unconnected.

How do you guys get around this? I guess children can be "virtually" connected with a dummy edge?

**Comment by darcynz** [comment]
I've now modelled the Virtual Chassis with an abstracted SwitchPort and it works okay, except where we have more than 4 members in a stack. It would be nice to be able to hide this to show the logical group, but I imagine thats not going to be straight forward.

**Comment by vrelk** [comment]
Yep. I have floating islands of switches because of this. It also makes a mess because if you have redundant links on 2 switches in a stack, it shows up as two different switches instead of combining it into one.

**Comment by noziwatele** [comment]
+1 for virtual chassis grouping. We use a lot of VSS/SVL and switch stacks and having individual devices displayed is not very useful or usable.

**Comment by dreng** [comment]
As there are reasons for grouping as well as against grouping, this should be an individual parameter.
